### PR TITLE
[docs][python] Misc doxygen improvements

### PR DESCRIPTION
**Pull request by enen92**
## Description
This PR provides a few cosmetic changes to the Python docs. Nothing special, just a few links, a few spaces to avoid line breaks to render on top of the revision tags, xbmc -> kodi, remove unicode references (python3), etc.

## Motivation and Context
Better docs

## How Has This Been Tested?
compile and see

## Types of change

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [x] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:

- [x] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [x] My change requires a change to the documentation, either Doxygen or wiki
- [x] I have updated the documentation accordingly
- [x] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [x] All new and existing tests passed

### CODE REVIEW

File: xbmc/interfaces/legacy/DrmCryptoSession.h [hunk 1]
```diff
@@ -121,11 +122,12 @@ namespace XBMCAddon
       /// @param      offlineKey      bool - Specifes the type of the request.
```
Comment (garbear) [inline xbmc/interfaces/legacy/DrmCryptoSession.h:122]: Misspelling: specifes -> specifies

I grepped the codebase, the is the only instance of `specifes`. Is there a plugin, similar to clang-tidy, that can check spelling? There's nothing sexier than pages and pages of endless documentation with no unsightly errors.

Comment (enen92) [reply xbmc/interfaces/legacy/DrmCryptoSession.h:122]: This was the better option I could find so far: https://github.com/louis-langholtz/pyspellcode

Comment (enen92) [reply xbmc/interfaces/legacy/DrmCryptoSession.h]: I gave the tool above a try but it didn't find any typos in the python docs. Even the "specifes" one was not found :/

**Review by phunkyfish — APPROVED** [review]
LGTM

File: xbmc/interfaces/legacy/DrmCryptoSession.h (hunk 1 above)
Comment (garbear) [reply xbmc/interfaces/legacy/DrmCryptoSession.h]: haha sucky

**Pull request merged**